### DISCUSSION TITLE
AI 也會做簡報！ChatBCG 一鍵幫你產生投影片

### DISCUSSION THREAD

**Discussion by awehao** [1101-GS3073-C/Python-Community-GS3073-C#571] [General]
看來期末專題沒時間作的人有救拉拉
更
謝謝同學熱烈捧場，我也去試了試，成果比想像中的好很多，不過還是要靠自己拉，不要被人工智慧打敗了，要激發自己的潛能，開發自己的大腦，才能像電影露西那樣，擁有超能力，知識就是力量，源源不斷的知識，就是源源不斷的力量，這是活著的意義，也是讓你能在社會上站穩腳步的重要基礎

https://www.inside.com.tw/article/30277-chatbcg

**Comment by number32984** [comment]
ai對不同人也會有不同的功能

**Reply by awehao** [reply]
這句話看似幹話，實則蘊含了相當深的涵義在，這位同學請問AI之於你有什麼功能呢?

**Comment by titan92601** [comment]
太神奇了吧

**Reply by awehao** [reply]
這位同學，您的這則回覆看似是紅色的，但是不論是好的回覆或是不那麼好的回覆，都是能夠促進我們這個社群，真的要感謝老師給了我們這個機會，才能在這討論，也才譨在這找到知己，對吧宗翰，我覺得我們不能因為加分與否就更改我們最初的想法，不過最後這一句是因為我看到沒有綠色才加的

**Comment by Chenszuhong** [comment]
哇 真是個好功能 但不知道做出來的實際情況如何 有機會試試看

**Comment by skyling0299** [comment]
透過玩chatGPT的時候發現，ai很吃重人下的指令。如果指令下不好他真得只會給垃圾出來，下得好才能拿到好的結果

**Reply by awehao** [reply]
這就稍微不方便之處，不過有心人肯定會好好下指令，不然成品跟他自己做出來的沒兩樣好像也沒什麼用

**Comment by brian431** [comment]
真是個有用的模型。好幾年過後很多作業大概都能用人工智慧來解決了，不知道之後的老師該怎麼出題目。

**Reply by chia-kai-chang** [reply]
我也希望大家能用AI寫作業，我會盡量出好題目給大家的(堅定)

**Comment by Xuewen628** [comment]
如果真的有人做出來會發大財吧 直接省去做簡報的時間

**Reply by chia-kai-chang** [reply]
簡報的重點是中心思想，是邏輯，是論述
以及根據論述而產生的流程圖或示意圖表
沒有論述的簡報，就算用AI美化，也是胡扯(這個字眼應該不算髒話，當老師要注意形象)

**Comment by Raywei0214** [comment]
我覺得我非常需要因為我現在正在做ppt

**Comment by awkward-willy** [comment]
簡報的製作真的很需要專業的技術，除了要做的有趣，內容豐富之外，也要兼顧報告的流暢性以及排版的美觀性等等，或許有了這樣的工具能讓大家在報告的時候有製作的典範，增進學習。

**Comment by awehao** [comment]
謝謝同學熱烈捧場，我也去試了試，成果比想像中的好很多，不過還是要靠自己拉，不要被人工智慧打敗了，要激發自己的潛能，開發自己的大腦，才能像電影露西那樣，擁有超能力，知識就是力量，源源不斷的知識，就是源源不斷的力量，這是活著的意義，也是讓你能在社會上站穩腳步的重要基礎

**Comment by bww0401** [comment]
剛剛才看到一篇一樣是ai可以幫忙寫論文的新聞，沒想到連ppt都可以吐出來，看來以後ai有可能還可以自己生成一部影片或電影，真的太有趣了，非常期待ai之後的發展

**Comment by larry40305** [comment]
我覺得這個真的推出後，希望某些教授或助教也能學著用這類的AI。有些PPT真的不是很方便看。

**Comment by magicmicheal** [comment]
這項功能能快速地讓PPT有個雛形，儘管最後要用人工的方式加上數據、圖片、流程圖，但已經省下了不少時間

**Comment by boson13579** [comment]
做簡報還是比較建議自己做。製作PPT中，查詢、整理、學習到的知識是ChatBCG沒辦法教你的，也你必須確保知識都能消化吸收，才可以將之轉換成更淺顯易懂的報告給老師及學生。

**Comment by Bonus502530** [comment]
我認為ChatBCG只能當成輔助製作簡報的工具，最好不能全部丟給AI做，就如同ChatGPT一樣，我有一門通識課的作業是申論題我曾經把題目丟上ChatGPT上，說然它可以回答出問題但是它給出來的答案其實還是跟上課講義上的內容不太一樣，我認為像這樣類似的AI還是參考就好。

**Comment by xCk27x** [comment]
### 【試著生成了我的期末報告XD】

<img width="859" alt="image" src="https://user-images.githubusercontent.com/90547641/211215844-557c9c6f-58a9-40db-81e9-252cc7886534.png">
# 我這篇給的指令是 "Please help me generate a report about how to use Python in Space Science and Satellite Technology"

> 必須說完全沒有幫助哈哈 
基本上就是依照一個簡單的模板 從輸入的內容抓取關鍵字 再用關鍵字去找文章丟到簡報裡
有一些答非所問 而且也沒看見任何的簡報技巧==

-> 所以各位還是好好寫期末報告吧QQ

**Comment by 0212Jerry** [comment]
我自己其實蠻喜歡做投影片的，對我來說，做投影片就是在整理自己的思緒，並讓別人在第一瞬間看懂我的內容，這同時也在訓練編排以及美編的能力。

**Comment by Valentinaa0214** [comment]
近期的Chat系列機器人的功能都讓人大開眼界，感覺未來的教育環境與AI的結合會越來越密切，學生與老師們可能也要在課綱上做對應的調整，隨著科技相互進步。

**Comment by Vincent20030314** [comment]
現在科技的進步太讓人震驚了，沒想到連做報告都能用AI寫。
